subprojects/build-cache/src/main/java/org/gradle/caching/configuration/BuildCache.java

```diff
@@ -34,12 +34,12 @@
     void setEnabled(boolean enabled);
 
     /**
-     * Returns whether pushing to the build cache is enabled.
+     * Returns whether storing in build cache is enabled.
```

Comment:
I think "Return true if this build may store outputs in the build cache" would be even clearer.

Reply:
FTR, we use "Returns whether..." for `boolean`-returning methods in many places.